addons/account/models/account_account.py

```diff
@@ -858,6 +1089,262 @@ def get_import_templates(self):
     def _merge_method(self, destination, source):
         raise UserError(_("You cannot merge accounts."))
 
+    def action_merge(self):
```

Comment:
This function needs tests

Reply:
Split into (at least) 3 functions:
* check (and determine the target account)
* ask confirmation
* actual merge

Reply:
Can't a refactor of `base.partner.merge.automatic.wizard` be useful?
There should be a lot of code in common.